sdk/textanalytics/Azure.AI.TextAnalytics/src/AnalyzeActionsOperation.cs

```diff
@@ -115,6 +116,11 @@ public class AnalyzeActionsOperation : PageableOperation<AnalyzeActionsResult>,
         /// </summary>
         private bool? _showStats { get; }
 
+        /// <summary>
+        /// Represents the job Id the service assigned to the operation.
+        /// </summary>
+        private string _jobId { get; }
```

Comment:
Is there any reason not to use a `private readonly string` here instead of the private getter?   It's a minimal efficiency, but would bypass the need to generate and use a getter method rather than direct access.

Reply:
Updated it to readonly field.